lm_eval/models/gpt2.py

```diff
@@ -12,6 +40,11 @@ def __init__(
         subfolder=None,
         tokenizer=None,
         batch_size=1,
+        accelerate=False,
+        device_map=None,
+        max_memory=None,
+        skip_tokenizer=False,
+        offload_folder="./offload"
```

Comment:
Should also be exposed in the CLI.

Reply:
I wanted to keep them inside `model_args` so that we keep the simplicity of the main script, we would add 4 args that are related to accelerate support only

Reply:
Hum seems okay to me, if the arguments are optional all good as people that don't use accelerate won't need to set those values, and those that are, should.

Reply:
Great ok let me put them

Reply:
I have put them but I am afraid this will break the model initialization for other modules than HF models please see https://github.com/younesbelkada/lm-evaluation-harness/blob/30b6b1bd54b0f406507cc1cfac69e76f4a0551f2/lm_eval/base.py#L110